mw-webapp/src/styles/_variables.scss

```diff
@@ -40,3 +43,10 @@ $sidebar-button-offset: 5px;
 // Size
 $width-primary: 100%;
 $height-full: 100%;
+
+// Checkbox
```

Comment:
Can we group Size, Color and Font Size styles together on the bottom of the file? 
Now I see that they are starting to get lost among the component styles

Reply:
Or maybe vice versa to the very top

Reply:
I'll do it later in the task [#41](https://github.com/tritonJS826/masters-way/issues/41 )